doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -555,18 +555,26 @@ Libraries for networking
 
     * :c:struct:`nrf_cloud_obj` structure and functions for encoding and decoding nRF Cloud data.
     * :c:func:`nrf_cloud_obj_pgps_request_create` function that creates a P-GPS request for nRF Cloud.
+    * :ref:`lib_nrf_cloud_coap` library for accessing nRF Cloud services using CoAP.
```

Comment:
This item is needed here. I believe there is a separate changelog entry for this new library. Anyway, this library is not added to the nRF Cloud library.